just_audio_background/lib/just_audio_background.dart

```diff
@@ -358,17 +351,17 @@ class _PlayerAudioHandler extends BaseAudioHandler
   List<MediaItem>? get currentQueue => queue.nvalue;
 
   _PlayerAudioHandler(String playerId) {
-    _init(playerId);
+    unawaited(_init(playerId));
```

Comment:
Not a fan of `unawaited_futures`.

Reply:
This is actually `discarded_futures`, so I can add `.ignore()` instead if you wish, but that would discard possible unhandled future rejection.

Reply:
I've removed `unawaited` though.